src/Robo/Commands/Setup/SettingsCommand.php

```diff
@@ -49,6 +49,9 @@ public function generateSiteConfigFiles() {
       }
     }
 
+    // Generate hash file in salt.txt.
+    $this->invokeCommand('drupal:hash-salt:init');
```

Comment:
This probably works fine, but as a minor stylistic nitpick, it seems like the convention when calling a method in the same class is to invoke the method directly, i.e. `$this->hashSalt()` instead of `$this->invokeCommand('drupal:hash-salt:init');`

Reply:
I waffled on that question; couldn't decide if the better style was to call it directly or to go through the extra layer of abstraction of `invokeCommand.`

Reply:
Just food for thought, I think I read that if you call the method directly in the class (ie `$this->hashSalt();`) then the robo command hooks do not run. So for example, if I wanted to run some code everytime the hash salt was generated, I could not hook into the `$this->` version...